src/features/ci-cd/hooks/track-latest-release.ts

```diff
@@ -25,10 +25,10 @@ const updateLatestRelease = async (
 			$select: 'id',
 			$expand: {
 				owns__release: {
-					$select: 'id',
+					$select: 'id' as const,
 					$top: 1,
 					// the most recently started build should be the latest, to ignore variable build times
-					$orderby: { start_timestamp: 'desc' },
+					$orderby: { start_timestamp: 'desc' as const },
```

Comment:
In the places where we need two `as const` casts on the same query,  can't' we just do it once in the whole dict to reduce a bit the `as const` everywhere? 
```suggestion
	const appsToUpdate = await api.get({
		resource: 'application',
		options: {
			$select: 'id',
			$expand: {
				owns__release: {
					$select: 'id',
					$top: 1,
					// the most recently started build should be the latest, to ignore variable build times
					$orderby: { start_timestamp: 'desc' },
					$filter: trackableReleaseFilter,
				},
			},
			$filter: {
				should_track_latest_release: true,
				owns__release: {
					$any: {
						$alias: 'r',
						$expr: {
							r: {
								id: { $in: releaseIds },
							},
						},
					},
				},
			},
		} as const,
	});
```

Reply:
Yeah we can, this was primarily a way of showing specifically what is causing the need for the `as const`